pkg/i2gw/providers/kong/storage.go

```diff
@@ -17,16 +17,19 @@ limitations under the License.
 package kong
 
 import (
+	kongv1beta1 "github.com/kong/kubernetes-ingress-controller/v2/pkg/apis/configuration/v1beta1"
 	networkingv1 "k8s.io/api/networking/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
 type storage struct {
-	Ingresses map[types.NamespacedName]*networkingv1.Ingress
+	Ingresses    map[types.NamespacedName]*networkingv1.Ingress
+	TCPIngresses []kongv1beta1.TCPIngress
 }
 
-func newResourcesStorage() *storage {
+func newStorage() *storage {
```

Comment:
nit: any reason for having a different function name than other implementations?

Reply:
Changed to be coherent with others.